### PR TITLE
Release core-react v0.6.0 & tokens v0.4.0

### PR DESCRIPTION
**Pull request by mimarz**
Preparing new release

- [x] core-react v0.6.0
- [x] tokens v0.4.0
- [x] Update component status

### CODE REVIEW

**Review by pomfrida — APPROVED** [review]
🎉

File: libraries/tokens/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.0] - 2020-09-02
+
+### Added
+
+#### Color
+
+Added missing overlay color `pressed_overlay_light`
```
Comment (vnys) [inline libraries/tokens/CHANGELOG.md:14]:
```suggestion
- Added missing overlay color `pressed_overlay_light`
```

File: libraries/tokens/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+
+### Removed
+
+Removed unused colors in EDS
```
Comment (vnys) [inline libraries/tokens/CHANGELOG.md:18]:
```suggestion
- Removed unused colors in EDS
```

File: libraries/tokens/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+
+- `field__fill_resting`
```
Comment (vnys) [inline libraries/tokens/CHANGELOG.md:20]:
```suggestion
  - `field__fill_resting`
```

File: libraries/tokens/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+- `field__fill_hover`
```
Comment (vnys) [inline libraries/tokens/CHANGELOG.md:21]:
```suggestion
  - `field__fill_hover`
```

File: libraries/tokens/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+- `field__fill_activated`
```
Comment (vnys) [inline libraries/tokens/CHANGELOG.md:22]:
```suggestion
  - `field__fill_activated`
```

File: libraries/tokens/CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+
+#### Color
+
+Changed `link_in_snackbars` for better contrast.
```
Comment (vnys) [inline libraries/tokens/CHANGELOG.md:28]:
```suggestion
- Changed `link_in_snackbars` for better contrast.
```

**Review by vnys — CHANGES_REQUESTED** [review]
Remember to bump the version of the storybook to the same version as the react components. And use lists in the changelog even though there’s only one item.

**Review by vnys — APPROVED** [review]
LGTM 👍

**Pull request merged**